File: ImportGLX/src/dto/RouteDto.ts
import IGlxDto from './interface/IGlxDto'

interface INodeData {
    nodeId: String
    distance: Number
    duration: Number
}

export default class RouteDTO implements IGlxDto {
    system_id: String
    glx_id: String
    data: {
        isReinforcementRoute: boolean
        isEmptyRoute: boolean
        routeNodes: INodeData[]
    }
    status: String = "Not Processed"
}

File: ViagemMasterData/DTOs/RedeMasterDataDTOs/RouteDTO.cs
using System;
using System.Collections.Generic;

namespace ViagemMasterData.DTOs.RedeMasterDataDTOs
{
    public class RouteDTO
    {
        public string _id { get; set; }
        public int distance { get; set; }
        public int duration { get; set; }
        public List<RouteNodesDTO> routeNodes { get; set; }
    }

    public class RouteNodesDTO
    {
        public NodeDTO nodeId { get; set; }
        public int distance { get; set; }
        public int duration { get; set; }
    }

    public class NodeDTO
    {
        public string _id { get; set; }
    }
}
